copado-function/app/Deploy.js

```diff
@@ -205,7 +204,7 @@ async function run() {
         Log.info('Merge into promotion branch');
         Log.info('===================');
         Log.info('');
-        Deploy.promote(CONFIG.toBranch, CONFIG.promotionBranch);
+        Deploy.promote(CONFIG.mainBranch, CONFIG.sourceBranch);
```

Comment:
promote only takes 1 parameter at the moment. (this was wrong before...)
pls fix

Reply:
fixed that, and changed the name of the variables inside the functions (the ones received as parameters) as well, pls check it

Reply:
much appreciated!